src/phoenix/server/api/routers/auth.py

```diff
@@ -187,4 +207,83 @@ async def refresh_tokens(request: Request) -> Response:
     return response
 
 
+@router.post("/initiate-password-reset")
+async def initiate_password_reset(request: Request) -> Response:
+    sender: EmailSender = request.app.state.email_sender
+    if sender is None:
+        raise HTTPException(
+            status_code=HTTP_503_SERVICE_UNAVAILABLE,
+            detail="SMTP unavailable",
+        )
+    assert isinstance(token_expiry := request.app.state.password_reset_token_expiry, timedelta)
+    data = await request.json()
+    if not (email := data.get("email")):
+        raise HTTPException(
+            status_code=HTTP_422_UNPROCESSABLE_ENTITY,
+            detail="Email required",
+        )
+    async with request.app.state.db() as session:
+        user = await session.scalar(_select_active_user().filter_by(email=email))
+    if user is None:
```

Comment:
Probably also need a check for the auth method (about to add OAuth). If the user is not local auth, we should raise an error message asking them to reset their password via their IDP.